org-brain.el

```diff
@@ -62,24 +64,94 @@ This will be used by `org-brain-new-child'."
   :group 'org-brain
   :type '(string))
 
+;;; Utils
+(defun flatten (obj)
```

Comment:
Not sure about praxis here, but usually functions defined by the package should start with `org-brain-`, but this is a utility function. Is this copied from another package (seems flatten is a pretty common name)? I'm mostly concerned with clashes regarding symbol names.